Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -91,6 +126,13 @@ class BRBViewController: UIViewController {
         
         state = newState
     }
+    
+    func showErrorAlert(error: String) {
+        let errorAlert = UIAlertController(title: error, message: "Unable to fetch new BRB account data. Please check your connection and try again.", preferredStyle: UIAlertControllerStyle.alert)
```

Comment:
Same thing about the type inference thing here. We can use `.alert` instead of `UIAlertControllerStyle.alert`